### PR TITLE
Spelling correction for Turkish translation

### PR DESCRIPTION
**Pull request by DursunCanPoyraz**
Spelling correction for Turkish translation

### CODE REVIEW

File: docs/tr/extend/testing.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# Testing
\ No newline at end of file
+# Testing
```
Comment (askvortsov1) [inline docs/tr/extend/testing.md:1]: Perhaps these page titles should be translated too, even if they're not populated yet?

**Review by askvortsov1 — CHANGES_REQUESTED** [review]

File: docs/tr/extend/testing.md (hunk 1 above)
Comment (DursunCanPoyraz) [reply docs/tr/extend/testing.md:1]: I do not know the Turkish of some titles. I will search and update.

Comment (askvortsov1) [reply docs/tr/extend/testing.md:1]: Thanks!

Comment (DursunCanPoyraz) [reply docs/tr/extend/testing.md]: @askvortsov1 I updated the titles.

**Review by askvortsov1 — APPROVED** [review]

**Review by askvortsov1 — APPROVED** [review]

**Review by tankerkiller125 — APPROVED** [review]

**Pull request merged**